pypeit/spectrographs/keck_nirspec.py

```diff
@@ -244,7 +313,7 @@ def lamps(self, fitstbl, status):
         raise ValueError('No implementation for status = {0}'.format(status))
 
     # TODO: This function is unstable to shape...
-    def bpm(self, filename, det):
+    def bpm(self, filename, det, shape=None):
```

Comment:
is this correct?

Reply:
I think it is now, see also analogous algorithm in spectrograph.